cls/SourceControl/Git/Settings.cls

```diff
@@ -171,6 +178,13 @@ ClassMethod Configure() As %Boolean [ CodeMode = objectgenerator ]
             do %code.WriteLine(" if value = ""System's Default Mode"" { set value = ""system""}")
             do %code.WriteLine(" elseif value = ""Yes"" { set value = 1}")
             do %code.WriteLine(" elseif value = ""No"" { set value = 0}")
+        } elseif ((propertyDef) && (propertyDef.Name = "environmentName")) {
+            do %code.WriteLine(" set list(1) = ""DEVELOPMENT""")
+            do %code.WriteLine(" set list(2) = ""TEST""")
+            do %code.WriteLine(" set list(3) = ""LIVE""")
```

Comment:
Per https://docs.intersystems.com/irislatest/csp/documatic/%25CSP.Documatic.cls?LIBRARY=%25SYS&CLASSNAME=%25SYSTEM.Version#SystemMode "FAILOVER" is also an option - should add that.